components/Home-components/about-the-clinic.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @next/next/no-img-element */
+import React from 'react'
+import { PageType } from '../types'
+import { Button } from '../data-components/button'
+import { WrapperLarge } from '../wrapper'
+
+export default function AboutTheClinic({ data }: PageType) {
+  return (
+    <section>
+      <WrapperLarge className="">
+        <div className="grid xl:grid-cols-2 gap-12 xl:gap-6 justify-items-center">
+          <div className="">
+            <img
+              src="/icons/about-clinic.png"
+              alt="אודות המרפאה"
+              className="justify-self-center"
+            />
+          </div>
+          <div className="grid place-items-center">
+            <div className="grid xl:place-items-start">
+              <div className="grid">
+                <a href="#" className="text-2xl text-header-blue xl:text-3xl">
```

Comment:
in nextjs links should have <Link> wrapping the <a>.
https://nextjs.org/docs/api-reference/next/link